Orange/tests/test_rules.py

```diff
@@ -36,12 +36,11 @@ def test_base_RuleLearner(self):
         raised).
         """
         base_rule_learner = _RuleLearner()
-        self.assertRaises(NotImplementedError, base_rule_learner.fit,
-                          self.iris.X, self.iris.Y)
+        #self.assertRaises(NotImplementedError, base_rule_learner.fit,
```

Comment:
If the code will not be used any more, just remove these lines. git remembers what was here :)